CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## HEAD
+
+* **tokens:** (breaking) update @mozilla-protocol/tokens to 5.0.2
```

Comment:
It would be great if you could add some update tips based on your experience updating Protocol to consume the new tokens. Something like:

- `$color-gray-` lighter than 50 can be swapped with their `$color-light-gray` equivalent. `$color-gray-` darker than 50 can be swapped with their `$color-dark grey` equivalent
- audit your use of red and yellow as the colours have gotten darker

I created a "migration tips" section for the 9.0 release but you could also just make them an indented list under this list item.

Reply:
Oh and a bullet for the break point change please 😬

Reply:
Willdo